src/api/dependency-discovery/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@senecacdot/satellite": "1.25.0"
+    "@senecacdot/satellite": "1.26.0"
```

Comment:
I just talked about this with Roxanne yesterday, I think Renovate can take care of updating Satellite if we make the version `^1.26.0` and override the range strategy for it to "[bump](https://docs.renovatebot.com/configuration-options/#rangestrategy)".

> bump = e.g. bump the range even if the new version satisfies the existing range, e.g. ^1.0.0 -> ^1.1.0

Reply:
Cool, someone should do a PR to make that happen.

Until then, this will get us on the latest version now across the board, and hopefully fix this crash case.

Reply:
@rclee91 would you mind filing that ^^ for follow-up?  It doesn't have to be you who fixes, but we should get it filed, or add that comment to an existing issue (we might have one?)